Google Play In-App Review Feature Request

## Feature Request:
Are there any plans to update the package to include the In App Review feature for Android? Currently the RequestReview() method works for iOS but not Android, and it appears that Google now has an API for it. Thank you!

https://developer.android.com/guide/playcore/in-app-review
https://developer.android.com/guide/playcore/in-app-review/kotlin-java#java

Thanks for all the advice. It takes more time to see if it really works due to an unexpected long publication time on the Play Store. I had to wait a few days to see the application in the internal channel. Anyway, it helped to resolve the first exception, but now I'm seeing a new exception:

```
java.lang.RuntimeException: Unable to instantiate activity ComponentInfo{com.appwithkiss.tipcalculator/com.google.android.play.core.common.PlayCoreDialogWrapperActivity}: java.lang.InstantiationException: java.lang.Class<com.google.android.play.core.common.PlayCoreDialogWrapperActivity> cannot be instantiated
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3228)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3443)
at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2040)
at android.os.Handler.dispatchMessage(Handler.java:107)
at android.os.Looper.loop(Looper.java:221)
at android.app.ActivityThread.main(ActivityThread.java:7520)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
Caused by: java.lang.InstantiationException: java.lang.Class<com.google.android.play.core.common.PlayCoreDialogWrapperActivity> cannot be instantiated
at java.lang.Class.newInstance(Native Method)
at android.app.AppComponentFactory.instantiateActivity(AppComponentFactory.java:95)
at androidx.core.app.CoreComponentFactory.instantiateActivity(Unknown Source:0)
at android.app.Instrumentation.newActivity(Instrumentation.java:1251)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3216)
```

--- a/README.md
+++ b/README.md
@@ -44,7 +44,7 @@ void OpenStoreReviewPage(string appId);
 ```
 
 #### Request Review
-UWP (all versions) and iOS only to prompt for the user to review the app. Only on iOS 10.3+ devices:
+UWP (all versions), Android and iOS only to prompt for the user to review the app. Only on Android 5.0 (API level 21) & iOS 10.3+ devices:
 Read: https://blog.xamarin.com/requesting-reviews-ios-10-3s-skstorereviewcontroller/
 
 ```csharp
@@ -53,6 +53,21 @@ Read: https://blog.xamarin.com/requesting-reviews-ios-10-3s-skstorereviewcontrol
 /// </summary>
 void RequestReview();
 ```
+
+### Android code shrinker (Proguard & r8)
+
+If you use the plugin with Link all, Release Mode and ProGuard/r8 enabled, you have to do the following:
+
+1. Create a `proguard.txt` file in your android project and add the following:
+
+```
+    -keep class com.google.android.play.core.common.PlayCoreDialogWrapperActivity
+    -keep class com.google.android.play.core.review.** { *; }
+    -keep class com.google.android.play.core.tasks.** { *; }
+```
+
+2. Include it to your project
+3. Properties > Build Action > ProguardConfiguration
 
 #### License
 Under MIT, see LICENSE file.

